src/format.rs

```diff
@@ -53,6 +53,9 @@ pub fn generic_args_to_string(args: &GenericArgs) -> String {
                 s.push_str(&type_to_string(output));
             }
         }
+        GenericArgs::ReturnTypeNotation => {
+            s.push_str("(..)")
+        }
```

Comment:
I added this path, but I think this path is never called, because `AngleBracketed.constraints` isn't handled.